services/packages/cleanup/cleanup.go

```diff
@@ -20,17 +20,25 @@ import (
 	debian_service "code.gitea.io/gitea/services/packages/debian"
 )
 
-// Cleanup removes expired package data
-func Cleanup(taskCtx context.Context, olderThan time.Duration) error {
-	ctx, committer, err := db.TxContext(taskCtx)
+// Task method to execute cleanup rules and cleanup expired package data
+func CleanupTask(ctx context.Context, olderThan time.Duration) error {
```

Comment:
So the manually click maybe conflict with the cron job? If two clean up run at the same time, what will happen?

Reply:
I would say it's not a problem because one of the transactions will fail. So only one task succeeds. We have that problem with all other cron jobs too I guess?